src/pow.cpp

```diff
@@ -24,8 +24,13 @@ unsigned int GetNextWorkRequired(const CBlockIndex* pindexLast, const CBlockHead
     // MVF-BU begin difficulty re-targeting reset (MVHF-BU-DES-DIAD-2)
     if (pindexLast->nHeight == FinalActivateForkHeight)
     {
-        LogPrintf("MVF FORK BLOCK DIFFICULTY RESET  %08x  \n", pindexLast->nBits);
-        return pindexLast->nBits;
+        arith_uint256 bnNew;
+
+        bnNew.SetCompact(0x207eeeee);
```

Comment:
This is more a note to remember discussions:

This parameter is hardcoded only temporarily here. The commented line below is to remind that we may change the reset difficulty to be a formula depending on difficulty at time of fork.